applications/trilinos_application/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -952,7 +1031,8 @@ class TrilinosBlockBuilderAndSolver
 
         //importing in the new temp vector the values
         int ierr = fixed.Import(fixed_local,dirichlet_importer,Insert);
-        if(ierr != 0) KRATOS_THROW_ERROR(std::logic_error,"Epetra failure found","");
+        if(ierr != 0)
```

Comment:
here it would be nice if you do 

KRATOS_ERROR << "Epetra failure found - error code = " << ierr;